lib/readline.js

```diff
@@ -1326,7 +1290,7 @@ exports.isFullWidthCodePoint = isFullWidthCodePoint;
 function codePointAt(str, index) {
   var code = str.charCodeAt(index);
   var low;
-  if (0xd800 <= code && code <= 0xdbff) { // High surrogate
+  if (code >= 0xd800 && code <= 0xdbff) { // High surrogate
```

Comment:
Yes, we can allow the yoda-style condition in range.
We can loosen the linter to use `exceptRange`.
`"yoda": [2, "never", { "exceptRange": true }]`